tests/test_scripts.py

```diff
@@ -11,8 +11,11 @@
 
 
 @pytest.mark.parametrize("test_name", TEST_NAMES)
-def test_script(test_name):
-    run = subprocess.run([os.path.join("tests", "scripts", "test_" + test_name + ".sh")])
+def test_script(test_name, cli_settings, tmp_path):
+    run = subprocess.run(
+        [os.path.realpath(os.path.join("tests", "scripts", "test_" + test_name + ".sh"))],
+        cwd=tmp_path,
```

Comment:
And this helps us to keep the pollution from some tests out of our git tree.

Reply:
Does pytest clean up the tmp_path if the test fails?

Reply:
pytest will keep the tmp_path for (defualt) 3 consecutive runs. Independent of failure.